src/proto/feature_flag.proto

```diff
@@ -3,6 +3,18 @@ option optimize_for = LITE_RUNTIME;
 
 package otbr;
 
+enum otbrProtoLogLevel
+{
+    PROTO_LOG_EMERG   = 0;  ///< System is unusable
```

Comment:
https://developers.google.com/protocol-buffers/docs/style#enums

The zero value enum should have the suffix `UNSPECIFIED`.

Reply:
👍